### PR TITLE
Remove autoyast clone button.

### PR DESCRIPTION
**Pull request by cwh42**

### CODE REVIEW

File: src/lib/installation/proposal_runner.rb [hunk 1]
```diff
@@ -654,21 +640,14 @@ def skip_buttons
 
     def build_dialog
       headline = @store.headline
+      change_point = Empty()
```
Comment (imobach) [inline src/lib/installation/proposal_runner.rb]: np: I would move this to the `else` of the following `if` condition. But it's just a matter of taste.

**Comment by imobach** [comment]
LGTM. I guessed you tested it manually, right? Maybe other teammate could think of some forgotten place. I can't :)

**Comment by lslezak** [comment]
After removing the `clone_finish` client you should remove it also from the [list of `*_finish` clients](https://github.com/yast/yast-installation/blob/master/src/lib/installation/clients/inst_finish.rb#L152).

**Comment by lslezak** [comment]
It seems you need to update the `Makefile.am` file as well: https://travis-ci.org/yast/yast-installation/builds/109902915#L1097

**Comment by cwh42** [comment]
@lslezak All green, now.

File: src/lib/installation/proposal_runner.rb (hunk 1 above)
Comment (cwh42) [inline src/lib/installation/proposal_runner.rb]: It actually was there before, but my personal taste told me to move it outside the condition.

**Comment by lslezak** [comment]
Thanks, LGTM

**Pull request merged**